lib/zendesk_call.rb

```diff
@@ -8,7 +8,7 @@ def call(email)
 class GetTicketRequest
   def call(email, user_id)
     @user_id = user_id
-    @tickets = client.search(:query => "#{email}")
+    @tickets = client.search(:query => "#{email}", :reload => 'true')
```

Comment:
Documentation had it listed as a string, which I thought was strange, but just mimicked it.